DatadogRUM/Sources/Debugging/RUMDebugging.swift

```diff
@@ -52,9 +49,8 @@ internal class RUMDebugging {
     }
 
     deinit {
-        let canvas = self.canvas
-        DispatchQueue.main.async {
-            canvas.removeFromSuperview()
+        DispatchQueue.main.async { [weak canvas] in
+            canvas?.removeFromSuperview()
```

Comment:
Rationale: calling `self.canvas` from deinit isn't safe as it may lead to calling UIKit APIs on background thread. This problem was surfaced in `testRandomlyCallingDifferentAPIsConcurrentlyDoesNotCrash()` test.